lib/chef/provisioning/transport/ssh.rb

```diff
@@ -91,6 +91,9 @@ def execute(command, execute_options = {})
         Chef::Log.debug("Stdout was:\n#{stdout}") if stdout != '' && !options[:stream] && !options[:stream_stdout] && config[:log_level] != :debug
         Chef::Log.info("Stderr was:\n#{stderr}") if stderr != '' && !options[:stream] && !options[:stream_stderr] && config[:log_level] != :debug
         SSHResult.new(command, execute_options, stdout, stderr, exitstatus)
+      rescue Exception => e
+        Chef::Log.error("Problem executing command on #{host} #{e}")
```

Comment:
Did this help with debugging?  I've started doing this pattern myself so I make sure the message is stored in a log file and not just on STDOUT